src/UI/Buyer/src/app/components/products/quantity-input/quantity-input.component.ts

```diff
@@ -109,6 +118,16 @@ export class OCMQuantityInput implements OnInit, OnChanges {
     if (!this.existingQty) {
       this.emit(this.form.get('quantity').value)
     }
+    if (this.groupedLis && this.li) {
```

Comment:
is this just groupedList appreciated? If so, just add the extra t.

If it's short for groupedLineItems, probably just change it to groupedLineItems or groupedList for ease of reading

Reply:
Updated to `groupedLineItems`